pkg/querier/chunk_store_queryable.go

```diff
@@ -103,6 +103,7 @@ func (s *chunkSeries) Iterator() storage.SeriesIterator {
 	return s.chunkIteratorFunc(s.chunks, model.Time(s.mint), model.Time(s.maxt))
 }
 
+// implements SeriesWithChunks
```

Comment:
Should this comment be with the declaration of `chunkSeries`?

Reply:
That is also a possibility. I don't have a strong preference where to put it, as long as it is documented.

Reply:
The declaration seems like a natural place to know all about the struct, which in this case the fact that `chunkSeries` implements `SeriesWithChunks`

Reply:
👍